src/ScoutServiceProvider.php

```diff
@@ -22,11 +23,21 @@ public function register()
     {
         $this->mergeConfigFrom(__DIR__.'/../config/scout.php', 'scout');
 
-        if (class_exists(MeiliSearch::class)) {
-            $this->app->singleton(MeiliSearch::class, function ($app) {
+        if (class_exists(MeilisearchClient::class)) {
```

Comment:
@mmachatschek, in this case, you are counting on the new meilisearch-php version (after the module case changes you did).

Should we add a `composer.json` protection? Or change the classes `MeilisearchClient` to the old ones?

Reply:
Or are my concerns not applicable here due to the PHP case-insensitiveness?

Reply:
Thanks @brunoocasali the case insensitivity wouldn't have complained about that but for consistency I updated the code 👍